edit/node.vue

```diff
@@ -0,0 +1,60 @@
+<script>
+import LabeledInput from '@/components/form/LabeledInput';
+import Labels from '@/components/form/Labels';
+import Taints from '@/components/form/Taints';
+import Footer from '@/components/form/Footer';
+import CreateEditView from '@/mixins/create-edit-view';
+import { DESCRIPTION, HOSTNAME } from '@/config/labels-annotations';
+
+export default {
+  name: 'EditNode',
+
+  components: {
+    Footer, LabeledInput, Labels, Taints
+  },
+
+  mixins:     [CreateEditView],
+
+  props: {
+    value: {
+      type:     Object,
+      required: true,
+    },
+  },
+
+  data() {
+    return {
+      DESCRIPTION,
+      HOSTNAME
+    };
+  },
+};
+</script>
+
+<template>
+  <div class="node">
+    <div class="row">
+      <div class="col span-6">
+        <LabeledInput v-model="value.metadata.labels[HOSTNAME]" :mode="mode" type="text" label="Hostname" placeholder="e.g. storage" />
```

Comment:
This isn't really a writable label; you can change it, but it's going to get overwritten by the kubelet to the original value in a few seconds when it next reports in.

Reply:
This is related to the conversation we had yesterday. I'm leaving it until I have an answer on what the appropriate field is. Interestingly I edited this value days ago and it's still my edit.

Reply:
Actually I decided to remove it all together since I'm switching to NameNsDescription. If the backend ends up supporting the field I'll add it back in.